sdk/include/sdk/DataPointValue.h

```diff
@@ -194,10 +198,42 @@ template <typename T> class TypedDataPointValue : public DataPointValue {
         return DataPointValue::operator==(other) && m_value == other.m_value;
     }
 
+    [[nodiscard]] std::string getValueAsString() const override;
+
 private:
     T m_value;
 };
 
+template <typename T> std::string TypedDataPointValue<T>::getValueAsString() const {
+    // Return empty string in case of unspecific data types
+    return "";
+}
+
+template <> inline std::string TypedDataPointValue<bool>::getValueAsString() const {
+    return std::to_string(static_cast<int>(m_value));
```

Comment:
Could be confused with numeric literals. Better to print `true` or `false`

Reply:
Not sure what is expected here. But I'll do.